lib/vector-config/tests/integration/smoke.rs

```diff
@@ -105,13 +105,13 @@ pub enum Encoding {
         ///
         /// If enabled, this will generally cause the output to be spread across more lines, with
         /// more indentation, resulting in an easy-to-read form for humans.  The opposite of this
-        /// would be the standard output, which eschews whitespace for the most succient output.
+        /// would be the standard output, which eschews whitespace for the most succinct output.
         pretty: bool,
     },
 
     #[configurable(description = "MessagePack encoding.")]
     MessagePack(
-        /// Starting offset for fields something something this is a fake description anyways.
+        /// Starting offset for fields something this is a fake description anyways.
```

Comment:
sometimes repeated random text is valuable, here it doesn't seem to be

Reply:
This one was probably intentional, in the sort of "this is a description that blah blah - it doesn't matter and is fake anyways" sort of way.

Reply:
Ah, it definitely was close to that. Had it been `blah blah` I would have assumed that. Certainly happy to 
drop/add to patterns (as I did for `Coc` and `many` here):
https://github.com/jsoref/vector/blob/e8475d01f8880d34b896a24379bab309fc091d7f/.github/actions/spelling/patterns.txt#L185